server/src/main/resources/application.yml

```diff
@@ -1,6 +1,6 @@
 tbb:
   cors:
-    urls: ${TBB_CORS_URLS:http://localhost:4200,http://localhost:4201,http://localhost:4202,https://tbbtalent-staging.herokuapp.com}
+    urls: ${TBB_CORS_URLS:http://localhost:4200,http://localhost:4201,http://localhost:4202,https://tctalent-test.org}
```

Comment:
Don't want to add tctalent-test.org - that will be in the staging JSON config.

Reply:
Removed the test URL